.drone.yml

```diff
@@ -20,6 +20,12 @@ steps:
     commands:
       - /root/.cargo/bin/cargo fmt -- --check
 
+  - name: dont pull in fat dependencies
+    image: ekidd/rust-musl-builder:1.51.0
+    commands:
+      - ! cargo tree -i zstd-sys
+      - ! cargo tree -i brotli-sys
```

Comment:
This fails in CI, but works correctly if I try it locally. Could be that drone doesnt support the ! operator (which inverts the program exit code).

Reply:
Should support it, it was most likely just a temporary network blip, as it passed fine on arm. I'm re-running it now.

Reply:
Something in the tree is still trying to pull zstd:

`error: package ID specification `zstd-sys` did not match any packages`

Reply:
That error means zstd is not in our dependencies, which is what we want. Try it on main, it on main, `cargo tree -i zstd-sys` gives exit code 0. [This is what I'm trying to do](https://stackoverflow.com/questions/367069/how-can-i-negate-the-return-value-of-a-process).